fhirstarter/fhirstarter.py

```diff
@@ -415,6 +415,19 @@ def openapi(self) -> dict[str, Any]:
 
             # Iterate over all operations for a given path
             for operation_name, operation in path.items():
+                # Skip operations that weren't created by FHIRStarter
+                operation_id = operation.get("operationId", "")
+                if not operation_id.startswith("fhirstarter|"):
+                    continue
+
+                # For operations that take a request body, change the application/json content type
+                # to application/fhir+json
+                if content := operation.get("requestBody", {}).get("content"):
+                    if "application/json" in content:
+                        content["application/fhir+json"] = content.pop(
+                            "application/json"
+                        )
```

Comment:
Swap out the request body content type of application/json for application/fhir+json.